examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,1090 @@
+      },
+      "source": [
+        "#### Batch Evaluator:\n",
+        "\n",
+        "Now that we have done FaithFulness and Relevancy Evaluation independently. LlamaIndex has `BatchEvalRunner` to compute multiple evaluations in batch wise manner."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 27,
+      "metadata": {
+        "id": "-t6Hxrc93jla"
+      },
+      "outputs": [],
+      "source": [
+        "from llama_index.evaluation import BatchEvalRunner\n",
+        "\n",
+        "# Let's pick top 10 queries to do evaluation\n",
+        "batch_eval_queries = queries[:10]\n",
+        "\n",
+        "# Initiate BatchEvalRunner to compute FaithFulness and Relevancy Evaluation.\n",
+        "runner = BatchEvalRunner(\n",
+        "    {\"faithfulness\": faithfulness_gpt4, \"relevancy\": relevancy_gpt4},\n",
+        "    workers=8,\n",
+        ")\n",
+        "\n",
+        "# Compute evaluation\n",
+        "eval_results = await runner.aevaluate_queries(\n",
+        "    query_engine, queries=batch_eval_queries\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 28,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "cAxrc5NF4T1r",
+        "outputId": "f80c105c-9d4b-4e10-8707-e4bad2bed9c0"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "1.0"
+            ]
+          },
+          "execution_count": 28,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "# Let's get faithfulness score\n",
+        "\n",
+        "faithfulness_score = sum(result.passing for result in eval_results['faithfulness']) / len(eval_results['faithfulness'])\n",
+        "\n",
+        "faithfulness_score"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 29,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "AGU3_QHW4ajS",
+        "outputId": "0e67a5f7-da94-40c4-8aa0-cd8874bb7ae9"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "1.0"
+            ]
+          },
+          "execution_count": 29,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "# Let's get relevancy score\n",
+        "\n",
+        "relevancy_score = sum(result.passing for result in eval_results['faithfulness']) / len(eval_results['relevancy'])\n",
+        "\n",
+        "relevancy_score\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vzjCqAIeRrk1"
+      },
+      "source": [
+        "There are many other different evaluation modules in LlamaIndex and you can check them [here](https://docs.llamaindex.ai/en/stable/module_guides/evaluating/root.html)"
```

Comment:
Add a conclusion header (and maybe 1 sentence overview). Otherwise, really like that you're linking out to more resources!

Reply:
Add conclusion header :)

Reply:
Sorry, Mis-understood the earlier comment. Added conclusion header now.